### PR TITLE
Setup basic express server

### PR DESCRIPTION
**Pull request by VladF146**
Installed express, dotenv, nodemon to create a simple, but functional express server.

When a request is being made to the root directory, the request path and method is logged to the server console and a JSON message is send back to the client.

### CODE REVIEW

**Review by dorosiemers — APPROVED** [review]

**Review by BektasTu — APPROVED** [review]
Top

**Review by nazardiachok — APPROVED** [review]

File: server.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require("dotenv").config();
+const express = require("express");
+
+const server = express();
+
+server.use((req, res, next) => {
+  console.log(req.method, req.path);
+  next();
+});
```
Comment (ThomasF85) [inline server.js]: This is useless for the app so far. Please remove useless code.

**Review by ThomasF85 — APPROVED** [review]
Approved, but please still delete the useless middleware function before merging.

File: server.js (hunk 1 above)
Comment (VladF146) [reply server.js]: done

**Review by kornrumpfl — APPROVED** [review]

**Pull request merged**